techniques/flash/FLASH35.html

```diff
@@ -25,7 +25,7 @@ <h3>A toggle button to pause and resume scrolling</h3>
     						to accomplish this. </p>
             </div>
          
-         <pre xml:space="preserve">import fl.accessibility.ButtonAccImpl;
+         <pre>import fl.accessibility.ButtonAccImpl;
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`